contracts/factories/DelegatedManagerFactory.sol

```diff
@@ -220,48 +222,62 @@ contract DelegatedManagerFactory {
      *
-     * NOTE: Modules must be passed before corresponding extensions in _initializeTargets otherwise initializeExtension will revert.
-     *
      * @param  _setToken                Instance of the SetToken
-     * @param  _ownerFeeSplit           Percent of fees in precise units (10^16 = 1%) sent to operator, rest to methodologist
+     * @param  _ownerFeeSplit           Percent of fees in precise units (10^16 = 1%) sent to owner, rest to methodologist
      * @param  _ownerFeeRecipient       Address which receives owner's share of fees when they're distributed
-     * @param  _initializeTargets       List of addresses of any extensions or modules which need to be initialized
+     * @param  _extensions              List of addresses of extensions which need to be initialized
      * @param  _initializeBytecode      List of bytecode encoded calls to relevant target's initialize function
      */
     function initialize(
         ISetToken _setToken,
         uint256 _ownerFeeSplit,
         address _ownerFeeRecipient,
-        address[] memory _initializeTargets,
+        address[] memory _extensions,
         bytes[] memory _initializeBytecode
     )
         external
     {
         require(initializeState[_setToken].isPending, "Manager must be awaiting initialization");
         require(msg.sender == initializeState[_setToken].deployer, "Only deployer can initialize manager");
-        _initializeTargets.validatePairsWithArray(_initializeBytecode);
-
-        for (uint256 i = 0; i < _initializeTargets.length; i++) {
-            address target = _initializeTargets[i];
-            require(!controller.isSet(target), "Target must not be SetToken");
-
-            // Because we validate uniqueness of _initializeTargets only one transaction can be sent to each module or extension during this
-            // transaction. Due to this no modules/extension can be used for any SetToken transactions other than initializing these contracts
-            target.functionCallWithValue(_initializeBytecode[i], 0);
-        }
+        _extensions.validatePairsWithArray(_initializeBytecode);
 
         IDelegatedManager manager = initializeState[_setToken].manager;
-        manager.updateOwnerFeeSplit(_ownerFeeSplit);
-        manager.updateOwnerFeeRecipient(_ownerFeeRecipient);
 
         // If the SetToken was factory-deployed & factory is its current `manager`, transfer
         // managership to the new DelegatedManager
         if (_setToken.manager() == address(this)) {
             _setToken.setManager(address(manager));
         }
 
-        manager.transferOwnership(initializeState[_setToken].owner);
-        manager.setMethodologist(initializeState[_setToken].methodologist);
+        for (uint256 i = 0; i < _extensions.length; i++) {
+            address extension = _extensions[i];
+            require(managerCore.isExtension(extension), "Target must be ManagerCore-enabled Extension");
```

Comment:
The targets of initialization in the `DelegatedManagerFactory` are now restricted to be `extensions` tracked on the `ManagerCore`.